firmware/controllers/system/timer/event_queue.cpp

```diff
@@ -100,10 +95,8 @@ efitime_t EventQueue::getNextEventTime(efitime_t nowX) const {
 			 *
 			 * looks like we end up here after 'writeconfig' (which freezes the firmware) - we are late
 			 * for the next scheduled event
-			 *
 			 */
-			efitime_t aBitInTheFuture = nowX + lateDelay;
-			return aBitInTheFuture;
+			return nowX + US2NT(10);
```

Comment:
no no no, event_queue.cpp operates in univeral X units not NT units

Reply:
Hmm.  In that case, the added `while (current->momentX > getTimeNowNt()) ;` is then technically invalid too.

Reply:
line 87 " * On this layer it does not matter which units are used - us, ms ot nt."